data-model/src/test/java/org/pdxfinder/graph/dao/TreatmentProtocolTest.java

```diff
@@ -0,0 +1,302 @@
+
+    private static final String CONTROL_DOSE_1 = "TEST_DOSE_1";
+    private static final String DRUG_DOSE_2 = "TEST_DOSE_2";
+    private static final String DRUG_DOSE_3 = "TEST_DOSE_3";
+
+    private List<TreatmentProtocol> treatmentProtocolList = new ArrayList<>();
+
+    @Before
+    public void InitModels() {
+
+        createMockDatasets1();
+        createMockDatasets2();
+    }
+
+    private void createMockDatasets1() {
+
+        List<Treatment> treatments;
+        treatments = createTreatmentsAndOntologies1();
+        createTreatmentComponentsAndProtocols1(treatments);
+    }
+
+
+    private List<Treatment> createTreatmentsAndOntologies1(){
+
+        Treatment treatment1 = new Treatment(FIRST_TREATMENT_NAME);
+        Treatment treatment2 = new Treatment(SECOND_TREATMENT_NAME);
+        Treatment treatment3 = new Treatment(THIRD_TREATMENT_NAME);
+
+        OntologyTerm ontologyTerm1 = new OntologyTerm(ONTOLOGY_URL, CONTROL_ONTOLOGY_LABEL);
+        OntologyTerm ontologyTerm2 = new OntologyTerm(ONTOLOGY_URL, DRUG_ONTOLOGY_LABEL);
+        OntologyTerm ontologyTerm3 = new OntologyTerm(ONTOLOGY_URL, DRUG_ONTOLOGY_LABEL_2);
+
+        TreatmentToOntologyRelationship ontologyRelationship1 = new TreatmentToOntologyRelationship();
+        TreatmentToOntologyRelationship ontologyRelationship2 = new TreatmentToOntologyRelationship();
+        TreatmentToOntologyRelationship ontologyRelationship3 = new TreatmentToOntologyRelationship();
+
+        ontologyRelationship1.setOntologyTerm(ontologyTerm1);
+        ontologyRelationship2.setOntologyTerm(ontologyTerm2);
+        ontologyRelationship3.setOntologyTerm(ontologyTerm3);
+
+        ontologyRelationship1.setTreatment(treatment1);
+        ontologyRelationship2.setTreatment(treatment2);
+        ontologyRelationship3.setTreatment(treatment3);
+
+        treatment1.setTreatmentToOntologyRelationship(ontologyRelationship1);
+        treatment2.setTreatmentToOntologyRelationship(ontologyRelationship2);
+        treatment3.setTreatmentToOntologyRelationship(ontologyRelationship3);
+
+        return Arrays.asList(treatment1, treatment2, treatment3);
+    }
+
+    private void createTreatmentComponentsAndProtocols1(List<Treatment> treatments) {
+
+        TreatmentProtocol treatmentProtocol = new TreatmentProtocol();
+        List<TreatmentComponent> componentsList= new ArrayList<>();
+
+        TreatmentComponent controlTC = new TreatmentComponent();
+        TreatmentComponent drug1 = new TreatmentComponent();
+        TreatmentComponent drug2 = new TreatmentComponent();
+
+        controlTC.setDose(CONTROL_DOSE_1);
+        drug1.setDose(DRUG_DOSE_2);
+        drug2.setDose(DRUG_DOSE_3);
+
+        controlTC.setDuration(CONTROL_DURATION_1);
+        drug1.setDuration(DRUG_DURATION_2);
+        drug2.setDuration(DRUG_DURATION_3);
+
+        controlTC.setTreatment(treatments.get(0));
+        controlTC.setType("Control");
+
+        drug1.setTreatment(treatments.get(1));
```

Comment:
Do you need to set the type on drug1 and drug2, or do they default to ...  not Control

Reply:
They default to Drug, but I'll explicitly set them for readability.